servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/events/fixtures/MockEventSubscriber.java

```diff
@@ -15,103 +15,19 @@
  */
 package org.projectnessie.server.events.fixtures;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.awaitility.Awaitility.await;
-
-import com.google.common.collect.ImmutableList;
-import java.time.Duration;
 import java.util.List;
-import java.util.concurrent.CopyOnWriteArrayList;
-import org.projectnessie.events.api.CommitEvent;
-import org.projectnessie.events.api.ContentRemovedEvent;
-import org.projectnessie.events.api.ContentStoredEvent;
 import org.projectnessie.events.api.Event;
-import org.projectnessie.events.api.MergeEvent;
-import org.projectnessie.events.api.ReferenceCreatedEvent;
-import org.projectnessie.events.api.ReferenceDeletedEvent;
-import org.projectnessie.events.api.ReferenceUpdatedEvent;
-import org.projectnessie.events.api.TransplantEvent;
 import org.projectnessie.events.spi.EventSubscriber;
-import org.projectnessie.events.spi.EventSubscription;
-
-public class MockEventSubscriber implements EventSubscriber {
-
-  private final List<Event> events = new CopyOnWriteArrayList<>();
-
-  private volatile boolean recording = false;
-
-  public List<Event> getEvents() {
-    return ImmutableList.copyOf(events);
-  }
-
-  public List<Event> awaitEvents(int numEvents) {
-    await()
-        .atMost(Duration.ofSeconds(5))
-        .untilAsserted(() -> assertThat(events).hasSize(numEvents));
-    return getEvents();
-  }
-
-  public void reset() {
-    events.clear();
-  }
-
-  public void startRecording() {
-    recording = true;
-  }
-
-  public void stopRecording() {
-    recording = false;
-  }
-
-  @Override
-  public void onSubscribe(EventSubscription subscription) {}
-
-  private void recordEvent(Event event) {
-    if (recording) {
-      events.add(event);
-    }
-  }
-
-  @Override
-  public void onReferenceCreated(ReferenceCreatedEvent event) {
-    recordEvent(event);
-  }
-
-  @Override
-  public void onReferenceUpdated(ReferenceUpdatedEvent event) {
-    recordEvent(event);
-  }
-
-  @Override
-  public void onReferenceDeleted(ReferenceDeletedEvent event) {
-    recordEvent(event);
-  }
 
-  @Override
-  public void onCommit(CommitEvent event) {
-    recordEvent(event);
-  }
+public interface MockEventSubscriber extends EventSubscriber {
```

Comment:
Turns out under `testFixtures` a bean is not detected by Quarkus, so I had to move it to `test` instead.